run-a-node.md

```diff
@@ -14,15 +14,16 @@ First off we install go 1.18.x
 
 1. `wget https://go.dev/dl/go1.18.2.linux-amd64.tar.gz`
 2. extract the runtime `sudo tar -C /usr/local -xzf go1.18.2.linux-amd64.tar.gz`
-3. Add go to your path (also add it to the end of your `.bashrc` file)
+3. Add go to your path
 
 ```
-export PATH=$PATH:/usr/local/go/bin
-export PATH=$PATH:$(go env GOPATH)/bin
+echo "export PATH=$PATH:/usr/local/go/bin" >> ~/.bashrc
+echo "export PATH=$PATH:$(go env GOPATH)/bin" >> ~/.bashrc
+. ~/.bashrc
 ```
 
-1. get the ignite cli `curl https://get.ignite.com/cli! | bash` (check out their site: https://docs.ignite.com/)
-2. check that ignite works `ignite version` - should produce output like
+4. get the ignite cli `curl https://get.ignite.com/cli! | sudo bash` (check out their site: https://docs.ignite.com/)
```

Comment:
This isn't necessarily required for all users, and is documented on Ignite's page too. Better to not use sudo when not needed. Maybe better to defer to the Ignite docs? 

Alternatively we update the build isntructions to check out a tag and use the Makefile, so that `kujirad version` works as expected

Reply:
Yeah I'm working on a more comprehensive update, thanks for the notes!